assets/js/middleware/store-api-nonce.js

```diff
@@ -39,12 +39,12 @@ const isStoreApiRequest = ( options ) => {
 const setNonce = ( headers ) => {
 	const nonce =
 		typeof headers?.get === 'function'
-			? headers.get( 'X-WC-Store-API-Nonce' )
-			: headers[ 'X-WC-Store-API-Nonce' ];
+			? headers.get( 'Nonce' )
+			: headers.Nonce;
```

Comment:
Ill double check this but I think it's the same (this will be an object). I'm pretty sure key=>value pairs are always objects. 

FYI it was prettier which changed this.

Reply:
In that case, let's 🚢 the change!

Reply:
> JavaScript arrays are not associative arrays and so, [array elements cannot be accessed using strings as indexes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#notes), but must be accessed using integers as indexes.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

I was correct, this is an object.